src/components/FocusZone/FocusZone.tsx

```diff
@@ -70,7 +70,12 @@ export class FocusZone extends BaseComponent<IFocusZoneProps, {}> implements IFo
 
     let parentElement = getParent(this.refs.root);
 
-    while (parentElement && parentElement !== document.body) {
+    while (
+      parentElement &&
+      parentElement !== document.body &&
+      parentElement.nodeType &&
```

Comment:
if you're comparing nodeType === 1, it is redundant to check for nodeType being truthy.